content/en/docs/making-requests.md

```diff
@@ -426,6 +426,195 @@ functional arguments to `NextPage`. For more information about overriding client
 
 ## Using Waiters
 
-{{% pageinfo info"info" %}}
-Waiters are not currently supported in the current {{% alias sdk-go %}} developer preview release.
-{{% /pageinfo %}}
+When interacting with AWS APIs that are asynchronous, you often need to wait 
+for a particular resource to become available in order to perform further 
+actions on it. 
+
+Let’s take an example of {{% alias service=DDBlong %}} `CreateTable` API. 
+After you invoke the API, the response returns immediately with a TableStatus 
+of CREATING, and you can’t invoke read or write operations until the table 
+status has been transitioned to `ACTIVE`. 
+
+Writing logic to continuously poll the table status can be cumbersome 
+and error-prone. This is especially true when you are dealing with multiple 
+AWS resources, as different resources often have different polling APIs and 
+success states that indicate the availability of that particular resource
+
+The waiters help take the complexity out of it and are simple APIs that handle 
+the polling task for you. It also provides polling configurations, such as the 
+maximum wait time, minimum delay and maximum delay between polling retries. 
+Waiters use a back-off strategy between each attempt. With the waiters utility, 
+you can focus on working with the resource without worrying how to make sure 
+that this resource is ready.
```

Comment:
This section seems a bit wordy and bit of an unnecessary lengthy of an explanation.